circle.yml

```diff
@@ -3,8 +3,14 @@ machine:
   php:
     # https://circleci.com/docs/environment#php
     version: 5.5.11
+  pre:
+    # @todo look for a better way to install phantomjs.
+    - sudo curl --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-2.1.1
```

Comment:
Thanks @joemiller, I was doing this install here because the phantomjs version pre-installed with CircleCI wasn't high enough for sitespeed.io so I was getting an error on `npm install -g sitespeed.io`. I have now moved the installation of sitespeed.io to package.json and it is also finding the correct version of phantomjs.